WooCommerce/Classes/ViewModels/Order Details/OrderDetailsPaymentAlerts.swift

```diff
@@ -0,0 +1,52 @@
+import UIKit
+import WordPressUI
+
+/// A layer of indirection between OrderDetailsViewController and the modal alerts
+/// presented to provide user-facing feedback about the progress
+/// of the payment collection process
+/// It is using a FancyAlertViewController at the moment, but this is the class
+/// to rewrite whenever we have the UI finalized.
+/// https://github.com/woocommerce/woocommerce-ios/issues/3980
+final class OrderDetailsPaymentAlerts {
+    private var alertController: FancyAlertViewController?
+    private var name: String?
+    private var amount: String?
+
+    func readerIsReady(from: UIViewController, name: String, amount: String) {
+        self.name = name
+        self.amount = amount
+
+        let newAlert = FancyAlertViewController.makeCollectPaymentAlert(name: name, amount: amount, image: .cardPresentImage)
```

Comment:
This alert doesn't follow the pattern of the alerts below. Please add a comment explaining why.